tests/client_coverage.rs

```diff
@@ -0,0 +1,160 @@
+    server.client().ca_details(&surplus).await.unwrap();
+    server.client().ca_delete(&surplus).await.unwrap();
+    server.client().ca_issues(&child).await.unwrap();
+    server.client().ca_history_commands(
+        &parent, None, None, None, None
+    ).await.unwrap();
+    server.client().ca_history_commands(
+        &parent, None, None, None, Some(Time::now())
+    ).await.unwrap();
+    server.client().ca_history_commands(
+        &parent, None, None, Some(Time::now()), Some(Time::now())
+    ).await.unwrap();
+    server.client().ca_history_details(
+        &parent, "0"
+    ).await.unwrap();
+    // ca_init_keyroll and ca_activate_keyroll tested in functional_keyroll
+    server.client().ca_update_id(&parent).await.unwrap();
+    server.client().ca_sync_parents(&child).await.unwrap();
+
+    server.client().child_connections(&parent).await.unwrap();
+    // child_add tested above
+    // child_import tested in functional_delegated_ca_import
+    server.client().child_details(&parent, &child.convert()).await.unwrap();
+    server.client().child_update(
+        &parent, &child.convert(), api::UpdateChildRequest::unsuspend()
+    ).await.unwrap();
+    server.client().child_contact(&parent, &child.convert()).await.unwrap();
+    // child_export tested in functional_delegated_ca_import
+    // child_delete tested in functional_parent_child
+    server.client().child_request(&child).await.unwrap();
+
+    server.client().parent_list(&parent).await.unwrap();
+    // parent_add tested above
+    server.client().parent_details(&parent, &testbed.convert()).await.unwrap();
+    // parent_delete tested in functional_parent_child
+
+    server.client().repo_request(&parent).await.unwrap();
+    server.client().repo_details(&parent).await.unwrap();
+    server.client().repo_status(&parent).await.unwrap();
+    // repo_update tested variously
+
+    let _ = server.client().roas_update(
+        &child,
+        api::RoaConfigurationUpdates::new(
+            vec![common::roa_conf("10.0.0.0/16-16 => 64496")],
+            vec![]
+        )
+    ).await.unwrap();
+    server.client().roas_list(&child).await.unwrap();
+    let update = api::RoaConfigurationUpdates::new(
+        vec![common::roa_conf("10.1.0.0/16-16 => 64496")],
+        vec![]
+    );
+    server.client().roas_dryrun_update(
+        &child, update.clone()
+    ).await.unwrap();
+    server.client().roas_try_update(
+        &child, update.clone()
+    ).await.unwrap();
+    server.client().roas_analyze(&child).await.unwrap();
+    server.client().roas_suggest(&child, None).await.unwrap();
+    server.client().roas_suggest(&child, Some(ca_res.clone())).await.unwrap();
+
+    server.client().bgpsec_add_single(
+        &child, common::asn("AS65000"),
+        rpki::ca::csr::BgpsecCsr::decode(
+            include_bytes!("../test-resources/bgpsec/router-csr.der").as_ref()
+        ).unwrap(),
+    ).await.unwrap();
+    server.client().bgpsec_list(&child).await.unwrap();
+
```

Comment:
Is `server.client().bgpsec_delete_single()` skipped because it's a variant of `bgpsec_add_single()` in that they both call `bgpsec_update()` ?

Reply:
Let’s go with “Yes” ;)

(I’ll add a comment.)